package.json

```diff
@@ -12,13 +12,14 @@
   "private": true,
   "dependencies": {
```

Comment:
Are you using `@bufbuild/connect`? It's mentioned as a runtime dependency in the [Buf docs](https://connect.build/docs/web/generating-code#local-generation).

Reply:
We don't directly import `@bufbuild/connect` anywhere. If this is a dep, it's a transitive dep of `@bufbuild/connect-web`. I don't think we need to worry about it.